index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css"/>
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <div class="container">
+    <div class="wrapper">
+        <header><h1>Alfred Mobile Portfolio</h1></header>
```

Comment:
- [ ] Please remove the `h1` tag from the header and add it in the headline section and put inside the header the following.
- The logo (if the design has one) of the design is on the left inside a link.
- The hamburger button is on the right and has a `nav` tag.

- [ ] Please put the proper background image (if the design has one).
- [ ] The header should has has the margins (top, bottom, left, right) defined in the design.